scout/adapter/mongo/query.py

```diff
@@ -148,6 +148,14 @@ def build_query(self, case_id, query=None, variant_ids=None, category='snv'):
                 ]
             })
 
+        if query.get('swegen') is not None:
+            mongo_query_minor.append({
+                '$or': [
+                    {'swegen': {'$exists': False}},
+                    {'swegen': {'$lt': query['swegen'] + 1}},
```

Comment:
Right; we do read a corresponding AF as well, but for 1000 individuals I thought this would be more straight forward (and correspond to the NGI count). Either goes; e.g. we might switch all to frq when we get a full local db in there.